### ISSUE TITLE
Layer normalization 

### ISSUE THREAD

**Issue by skaae** [Lasagne/Lasagne#738]
I'm implementing layernormalization from http://arxiv.org/pdf/1607.06450v1.pdf
 Should i create a pr for this to lasagne? 

The feedforward variant can be implemented as separate layer similar to `BatchNormalizationLayer` and the `batch_norm` helpder. 

For GRU and LSTM i think you would need a lot of if statements to handle layer normalization because the gate_biases are no longer used and because of the 'precompute' option.

**Comment by f0k** [comment]
Duplicate of #736. Short result: I'd suggest the feedforward implementation to be a superclass for `BatchNormalizationLayer`, to avoid code duplication. I wouldn't name it after the paper (`LayerNormalizationLayer`?), but in a way that conveys it's normalizing to zero mean and unit variance. Maybe `StandardizationLayer`, with a `layer_norm` helper?
